lib/cable_ready/operation_builder.rb

```diff
@@ -43,9 +43,13 @@ def add_operation_method(name)
           options.each { |key, value| options[key] = value.send("to_#{key}".to_sym) if value.respond_to?("to_#{key}".to_sym) }
         end
         options["selector"] = selector if selector && options.exclude?("selector")
-        options["selector"] = previous_selector if previous_selector && options.exclude?("selector")
+        if previous_selector && options.exclude?("selector")
+          options["selector"] = previous_selector
+          options["xpath"] = previous_xpath if previous_xpath
```

Comment:
I think I added that in because the output looked a bit weird otherwise. Namely; in the case where `previous_xpath` is `nil`, it would assign the `xpath` key to the payload, but with an undefined value. It felt like not assigning it at all and not increasing the payload size was preferable in that case? Like if `xpath` is `nil`, then the payload shouldn't have an `xpath` key at all instead of having an empty one? What do you think?